scripts/post-install.sh

```diff
@@ -60,31 +58,41 @@ fi
 # Distribution-specific logic
 if [[ -f /etc/redhat-release ]]; then
     # RHEL-variant logic
-    which systemctl &>/dev/null
-    if [[ $? -eq 0 ]]; then
-	    install_systemd
+    if pidof systemd &>/dev/null; then
```

Comment:
This should probably check for pid 1 instead. systemd could be running on the system, but not be the init manager. Such as if it's running within a container.
A safe test would be `if [[ "$(readlink /proc/1/exe)" == */systemd ]]; then`